crates/nu-command/src/filesystem/rm.rs

```diff
@@ -40,13 +40,9 @@ impl Command for Rm {
     }
 
     fn signature(&self) -> Signature {
-        let sig = Signature::build("rm")
+        Signature::build("rm")
             .input_output_types(vec![(Type::Nothing, Type::Nothing)])
-            .required(
-                "filename",
-                SyntaxShape::GlobPattern,
-                "The file or files you want to remove.",
-            )
+            .rest("paths", SyntaxShape::GlobPattern, "The file paths(s) to remove.")
```

Comment:
Do I read that right that this would now not complain about a missing path?

Maybe still worth raising an error at runtime. (Yes you could not completely freely run `rm ...$my_unknown_list`)
This would kind of give more peace of mind when running `rm` bare and you don't see any output.

Reply:
That would be a possibility, but somehow my taste develops in the direction that our internal commands should pull less tricks that you can not perform as a Nu library writer. 
But maybe your suggestion gets us closer to fulfilling the expectations from #11796

Reply:
Yeah, in the future we should consider adding the required rest parameter that I think you mentioned. Otherwise, I think this is the best of both worlds that is currently possible.

Reply:
Actually, it looks like the other filesystem commands error if the rest arg is empty. I'll just match their behavior for consistency.

Reply:
Thanks!